cf/service_broker.go

```diff
@@ -2,13 +2,18 @@ package cf
 
 import (
 	"context"
+
+	"github.com/Peripli/service-broker-proxy-cf/version"
 	"github.com/Peripli/service-broker-proxy/pkg/platform"
 	"github.com/cloudfoundry-community/go-cfclient"
 )
 
 // GetBrokers implements service-broker-proxy/pkg/cf/Client.GetBrokers and provides logic for
 // obtaining the brokers that are already registered at the cf.
 func (pc PlatformClient) GetBrokers(ctx context.Context) ([]platform.ServiceBroker, error) {
+	// Log the version regularly so it is not lost with old log messages
+	version.Log()
```

Comment:
Reconcilation job is not the right place to log version. The proxies have health apis similar to sm. Logging there would make more sense.